src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -342,7 +342,9 @@ def _detect_performance_problems(
     ]
 
     for detector in detectors:
-        run_detector_on_data(detector, data)
+        with sentry_sdk.start_span(op="function", description="run_detector_on_data") as span:
```

Comment:
can we name the detector on the span description instead

Reply:
(we can pull percentiles per detector span that way)

Reply:
@shruthilayaj In Metrics? In that case should I use a tag, so it can segment properly?

Reply:
nah, just with indexed data (at least we can check span summary that way and have a row per detector) - also just lets us read the span waterfall easier
if we want to pull metrics, we can have the span op be per detector - either works!

Reply:
👍🏻 ok